src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-failure.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: noticeNetworkFailure (iOS SDK API)
+type: apiDoc
+shortDescription: Manually record a failed transactional network request.
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - iOS SDK API
+metaDescription: New Relic API for iOS mobile app monitoring to manually record a failed transactional network request.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/notice-network-failure-ios-sdk-api
+---
+
+## Syntax
+
+```
++ (void)noticeNetworkFailureForURL:(NSURL*)url
+                        httpMethod:(NSString*)httpMethod
+                         withTimer:(NRTimer*)timer
+                    andFailureCode:(NSInteger)iOSFailureCode;
+```
+
+Manually record a failed transactional network request.
+
+## Requirements
+
+Compatible with all agent versions.
+
+## Description
+
+Failed requests are requests that fail to receive a complete response from
+the server due to, e.g., TCP timeouts, SSL failures, connection closures, etc.
+
+Refer to the NRNetworkFailureCode enum for failure type symbolic names.
```

Comment:
@cdillard-NewRelic Can you provide the correct docs links for both the NRNetworkFailureCode enum and Apple's documented NSURLConnection failure codes?

Reply:
Apples documented failure codes:  [URL Loading System Error Codes](https://developer.apple.com/documentation/foundation/1508628-url_loading_system_error_codes) 

I removed the reference to NRNetworkFailureCode as I think its just confusing and isn't required. The URL system errors from previous link are all that is needed for this API. Thank you!